app/src/main/java/org/mozilla/fenix/wallpapers/WallpapersUseCases.kt

```diff
@@ -485,6 +485,7 @@ class WallpapersUseCases(
 
         private fun selectWallpaper(wallpaper: Wallpaper) {
             settings.currentWallpaperName = wallpaper.name
+            settings.currentWallpaperTextColor = wallpaper.textColor ?: 0
```

Comment:
It would be good to verify this storage in a test IMO

Reply:
Added tests.